handbook/engineering/cloud/goals.md

```diff
@@ -0,0 +1,69 @@
+# Cloud goals and priorities
+
+## Guiding principles
+
+1. **Make it work**: Build the backend blocking work, and expose it (even in a roughly usable way) to the Sourcegraph team. We will be able to quickly surface any glaring issues and will have more thoughts around usability. Take shortcuts where possible (this is currently due to the team having fewer frontend resources).
+   - Where possible, making it work and usable should be combined (avoid duplicate efforts), but if it's possible to separate the two in order to move things forward, we should!
+1. **Make it usable**: The experience has been designed and thought through. We feel good about putting this in front of users, and they will find it valuable!
```

Comment:
(I know this got moved but worth changing now)

I don't think "usable" is a high enough bar. Please change this to "polished" or something else more ambitious.

```suggestion
1. **Make it polished**: The experience has been designed and thought through. We feel good about putting this in front of users, and they will find it valuable!
```

Reply:
What we're trying to communicate here is a good user experience. Polished does not necessarily mean good UX and can also create the impression of over doing it. I also see how make it usable does not make a high enough bar. I'm curious if @quinnkeast has an idea about how to best capture this.

Reply:
> Polished does not necessarily mean good UX

It does to me!

> can also create the impression of over doing it.

Yeah, don't want that.


What about "smooth"?

```suggestion
1. **Make it smooth**: The experience has been designed and thought through. We feel good about putting this in front of users, and they will find it valuable!
```